packages/cicero-core/src/metadata.js

```diff
@@ -316,6 +344,42 @@ class Metadata {
         return this.packageJson.name + '@' + this.packageJson.version;
     }
 
+    /**
+     * Returns the dimension of the image
+     * @param {Buffer} buffer buffer of the image
+     * @param {String} mimeType the mime-type of image
+     * @returns {Object} the dimension of the image
+     */
+    static getImageDimensions(buffer, mimeType) {
+        if(mimeType === 'image/png') {
+            return {
+                height: buffer.readUInt32BE(20),
+                width: buffer.readUInt32BE(16)
+            };
+        }
+        throw new Error('dimension calculation not supported for this file');
+    }
+
+    /**
+     * @param {Number} height height of the image
+     * @param {Number} width width of the image
+     * @param {Number} epsilon allowed error in dimensions of the image file
+     * @returns {Boolean} whether the dimensions are vaild
+     */
+    static areDimensionsAllowed(height, width, epsilon=4) {
```

Comment:
Seeing this code, I think I agree with Dan. We don't know what the requirements will be down the road: should epsilon be allowed at all, should it be as a percentage of image size, etc. And we have to have tests for this code, which are unnecessary at the moment.

I would just remove it or replace it by a strict equality.

The size should be exactly 128x128.